benchmarks/asv.conf.json

```
@@ -113,15 +113,15 @@
     "include": [
         // minimum supported versions
         {
-            "python": "3.7",
+            "python": "3.8",
             "build": "",
             "numpy": "1.17.5",
             "pandas": "1.3.0",
             "scipy": "1.6.0",
             // Note: these don't have a minimum in setup.py
             "h5py": "3.1.0",
-            "ephem": "3.7.6.0",
-            "numba": "0.40.0"
+            "ephem": "3.7.7.0",  // first version to support py 3.8
```

```suggestion
            "ephem": "3.7.7.1",  // first version to support py 3.8
```
If I understood the information on pypi correctly.

---

Version 3.7.7.0 has wheels available for python 3.8: https://pypi.org/project/ephem/3.7.7.0/#files

That's how I identified these versions.  Is there a different/better way?

---

You're asking the wrong person!  I was just browsing to get a sense for the consequences.